server/command/install.go

```diff
@@ -11,36 +11,81 @@ import (
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
 	"github.com/mattermost/mattermost-plugin-apps/server/httpin/dialog"
-	"github.com/mattermost/mattermost-plugin-apps/server/utils"
 )
 
-func (s *service) executeInstall(params *params) (*model.CommandResponse, error) {
+func (s *service) executeInstallMarketplace(params *commandParams) (*model.CommandResponse, error) {
```

Comment:
maybe add a comment for future readers that for this path (Marketplace) we've previously added the manifests to the store (using `InitGlobal`?).